test/cpp/api/serialize.cpp

```diff
@@ -54,7 +54,7 @@ TEST(SerializeTest, BasicToFile) {
 
   auto x = torch::randn({5, 5});
 
-  torch::test::TempFile tempfile;
+  auto tempfile = torch::make_tempfile();
```

Comment:
Do we really want this to appear in the top namespace? Seems like a namespace pollution issue to me.

Reply:
I'll put it into `utils`